ner_v2/detectors/numeral/number/zh-TW/number_detection.py

```diff
@@ -0,0 +1,162 @@
+from __future__ import absolute_import
+import os
+
+try:
+    import regex as re
+
+    _re_flags = re.UNICODE | re.V1 | re.WORD
+except ImportError:
+    import re
+
+    _re_flags = re.UNICODE
+
+from ner_v2.constant import LANGUAGE_DATA_DIRECTORY
+from ner_v2.detectors.numeral.constant import NUMBER_DETECTION_RETURN_DICT_SPAN, \
+    NUMBER_DETECTION_RETURN_DICT_UNIT, NUMBER_DETECTION_RETURN_DICT_VALUE
+from ner_v2.detectors.numeral.number.standard_number_detector import BaseNumberDetector
+
+from chatbot_ner.config import ner_logger
+
+
+"""
+mapping some special character for chinese (traditional)
+use to replace in text string
+"""
+special_chars_mapping = {
+    ',' : '、', # comma character
+    '.' : '點' #dian ( period )
```

Comment:
E203 whitespace before ':'
E261 at least two spaces before inline comment
E262 inline comment should start with '# '